docs/reference/data-frames/apis/preview-transform.asciidoc

```diff
@@ -40,6 +49,13 @@ Previews a {dataframe-transform}.
 `pivot` (Required)::
   (object) Defines the pivot function `group by` fields and the aggregation to
   reduce the data. See <<data-frame-transform-pivot>>.
+  
+[discrete]
+[[preview-data-frame-transform-response]]
+==== {api-response-body-title}
+
+`preview`::
+  (array) An array of {dataframe} objects.
```

Comment:
@hendrikmuhs Is this an appropriate description? If not, what would you recommend calling the contents of the array?

Reply:
`An array of documents` is more apt. 

They are the JSON representation of the documents that will be indexed into the destination of the transform. They are not {dataframe} objects.

Reply:
Thanks @benwtrent I've revamped the description